src/testing/workspace-fixtures.ts

```diff
@@ -96,6 +97,11 @@ export const defaultGoogleWorkspace: GoogleWorkspace = {
   policies: [],
 };
 
+export const defaultGoogleWorkspaceWrapper: InitializedWorkspaceWrapper = {
```

Comment:
Could you call this `defaultInitializedGoogleWorkspace`? The name "wrapper" doesn't make sense to me here, and the only thing that differs from `defaultGoogleWorkspace` is the initialized bit.